ci-jobs/scripts/start-emulator.sh

```diff
@@ -3,6 +3,10 @@
 # This script was copy-pasted from https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/android?view=azure-devops#test-on-the-android-emulator
 # with some changes
 
+# Use Java 8 for these tasks, or it will error with
+# Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlSchema
+export JAVA_HOME=`/usr/libexec/java_home -v 1.8`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/dpgraham/appium-espresso-driver/pullRequest?prid=5175446)